### ISSUE TITLE
Scrolling flatlist with talkback on android does not announce how far you've scrolled the list

### ISSUE THREAD

**Issue by eivisand** [facebook/react-native#30754]
## Description
With a vertical scrollable list in android I expect talkback to announce "In List showing item n to m of N" when doing a scroll gesture. However nothing is announced. On iOS voice over announces "Page n of N".

## React Native version:
System:
    OS: macOS 11.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
    Memory: 68.12 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 12.19.0 - /var/folders/65/tz74fzb53c52lmnp6hd4q74r0000gn/T/yarn--1610963702428-0.7934317913162223/node
    Yarn: 1.22.10 - /var/folders/65/tz74fzb53c52lmnp6hd4q74r0000gn/T/yarn--1610963702428-0.7934317913162223/yarn
    npm: 6.14.8 - ~/.nvm/versions/node/v12.19.0/bin/npm
    Watchman: Not Found
  Managers:
    CocoaPods: 1.10.0 - /Users/me/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.2, DriverKit 20.0, macOS 11.0, tvOS 14.2, watchOS 7.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.6953283
    Xcode: 12.2/12B45b - /usr/bin/xcodebuild
  Languages:
    Java: 14.0.1 - /usr/bin/javac
    Python: 2.7.16 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: Not Found
    react-native: 0.62.2 => 0.62.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

## Steps To Reproduce
1. Have an app with a scrollable flatlist of items
2. Run your app in android
3. Turn on talkback (settings>accessibility>talkback)
4. Go to your app with the list
5. Scroll the list (with one finger swipe from side to side once)

## Expected Results
 I expect talkback to announce "In List showing item n to m of N" when doing a scroll gesture

## Snack, code example, screenshot, or link to a repository:
https://snack.expo.io/QGs1YazZz

**Comment by blavalla** [comment]
This has been fixed by https://github.com/facebook/react-native/pull/33180. Closing out.

**Comment by necolas** [comment]
It looks like that patch was reverted in April. Couldn't tell why or if an updated version was merged at a later date
